src/components/Chat.astro

```diff
@@ -73,6 +80,42 @@ export const prerender = false;
     }
   }
 
+  // The Rive animation
+  const luna = new rive.Rive({
+    src: "/luna.riv",
+    canvas: document.getElementById("canvas") as HTMLCanvasElement,
+    autoplay: true,
+    stateMachines: "State Machine 1",
```

Comment:
I would rename the state machine to something a bit more descriptive, which could be as simple as `default-state-machine`